versioned/tiered/mongodb/src/main/java/com/dremio/nessie/versioned/store/mongodb/MongoSerDe.java

```diff
@@ -45,31 +49,29 @@
 @SuppressWarnings({"unchecked", "rawtypes"})
 final class MongoSerDe {
   private static final Map<ValueType<?>, Function<BsonWriter, MongoBaseValue>> CONSUMERS;
-  private static final Map<ValueType<?>, Map<String, BiConsumer<BaseValue, BsonReader>>> DESERIALIZERS;
-
-  private static final String MONGO_ID_NAME = "_id";
+  private static final Map<ValueType<?>, BiConsumer<Document, BaseValue>> DESERIALIZERS;
```

Comment:
A common pattern for this type of thing is just put the construction in the declaration as opposed to having a separate static block. Makes things cleaner for simple constructions like those here.

private static final Map<A, B> FOO = ImmutableMap.builder()
  .put(...,...)
  .put(...,...)
  .put(...,...)
  .build();